spec/support/sorcery_login.rb

```diff
@@ -0,0 +1,13 @@
+module Sorcery
+  module TestHelpers
+    module Rails
+      def login_user_post(user, password)
+        page.driver.post(new_user_session, { username: user, password: password})
```

Comment:
Line is too long. [81/80]<br>Redundant curly braces around a hash parameter.<br>Space inside } missing.